src/views/AnnualReport.vue

```diff
@@ -106,9 +106,9 @@
                 <h1 id="annual-report-header">
                   File {{ ARFilingYear }} Annual Report
                   <span
-                    v-if="getReportState"
+                    v-if="filingId > 0"
                     class="font-italic"
-                  > &mdash; {{ getReportState }}</span>
+                  > &mdash; Draft</span>
```

Comment:
The only thing we need to know is this report's state, which can only be "Draft".

If filing id = 0 then this is a new filing. If it's > 0 then it's a draft that we're continuing to work on.

Reply:
In Dev - new filing:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/7d5651d1-4b20-4303-a1ea-cade7b8fea5a)

in Dev - draft filing:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/7748a97a-545f-496f-bf93-66ee096b57a9)

In this PR - new filing:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/a8fa258f-869e-4cd0-b8bd-3c7fedb829f6)

In this PR - draft filing:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/dc15d898-1e36-441c-96be-cc42fe76d27d)